src/boot/bootctl.c

```diff
@@ -52,6 +52,9 @@ static bool arg_print_dollar_boot_path = false;
 static bool arg_touch_variables = true;
 static PagerFlags arg_pager_flags = 0;
 static bool arg_graceful = false;
+static enum { MID_NO, MID_YES, MID_AUTO, } arg_make_mid = MID_AUTO;
```

Comment:
please, no needless abbreviations, and there's no shame in line breaks. Moreover "mid" as short for "machine id" is a bit of a misnomer, since this reads as if you'd make the machine ID itself somehow, but it's the *directory* named after it you are making.

maybe:

```c
static enum {
        MAKE_MACHINE_ID_DIRECTORY_NO,
        MAKE_MACHINE_ID_DIRECTORY_YES,
        MAKE_MACHINE_ID_DIRECTORY_AUTO,
} arg_make_machine_id_directory = MAKE_MACHINE_ID_DIRECTORY_AUTO;
```

THat said, we often use `tristates` for cases like this. i.e. if you have something that is mostly a boolean, but has one extra state that is omething like "unset", "does not apply", "auto" or so, then we use `int` as type and assign > 0 for true, == 0 for false, and < 0 for that special unset/auto/na case. Thus I'd probably just drop the enum here, and use one of these tristates, i.e. change the above line to:

```c
static int arg_make_machine_id_directory = -1;
```

Reply:
Replaced with `int` that uses -1 for FILE_NOT_FOUND, as suggested.